app/src/main/java/edu/grinnell/grinnell_publications_android/Models/Realm/RealmStory.java

```diff
@@ -12,95 +12,133 @@
  * @see RealmObject
  */
 public class RealmStory extends RealmObject implements Story{
-    private RealmPublication publication;
-    private String publicationDate;
-    private String lastUpdated;
-    private RealmList<RealmAuthor> author;
-    private RealmReactions reactions;
-    private String fullText;
-    private String blurb;
+    private String datePublished;
+    private String brief;
+    private String headerImage;
+    private String publication;
+    private String dateEdited;
+    private int articleId;
     private String title;
-    private String thumbnailUrl;
+    private String content;
+    private RealmList authors;
 
 
     /* Default constructor for Realm */
     public RealmStory(){}
 
-    public RealmStory(RealmPublication publication, String publicationDate, String lastUpdated,
-                      RealmList<RealmAuthor> author, RealmReactions reactions, String fullText,
-                      String blurb, String title, String thumbnailUrl) {
+    public RealmStory(String datePublished, String brief, String headerImage, String publication,
+                      String dateEdited, int articleId, String title, String content,
+                      RealmList authors) {
+        this.datePublished = datePublished;
+        this.brief = brief;
+        this.headerImage = headerImage;
         this.publication = publication;
-        this.publicationDate = publicationDate;
-        this.lastUpdated = lastUpdated;
-        this.author = author;
-        this.reactions = reactions;
-        this.fullText = fullText;
-        this.blurb = blurb;
+        this.dateEdited = dateEdited;
+        this.articleId = articleId;
         this.title = title;
-        this.thumbnailUrl = thumbnailUrl;
+        this.content = content;
+        this.authors = authors;
     }
 
     /** Setters */
-    public void setPublication(RealmPublication publication){
+    public void setPublication(String publication) {
+        if (publication.isEmpty() || publication == null) {
+            return;
+        }
         this.publication = publication;
     }
-    public void setPublicationDate(String publicationDate){
-        this.publicationDate = publicationDate;
+
+    public void setDatePublished(String datePublished) {
```

Comment:
Please document all methods you make public so the client who calls this method has an easier time knowing what it does, what the parameter are and what they mean. When someone calls this method, they can just hover over it and see the documentation rather than having to go to 

A good example is @as-f's documentation of his public methods here: https://github.com/GrinnellAppDev/DB-Android/pull/3/files#diff-23c4fc42ab674199182b5a617e912235